python/tk_3dsmaxplus/menu_generation.py

```diff
@@ -14,6 +14,7 @@
 import os
 import sys
 import traceback
+import unicodedata
```

Comment:
Is this import needed? It's added here, but no other changes were made to this file.